public/index.html

```diff
@@ -28,7 +28,11 @@
     <script src="https://kit.fontawesome.com/2475edd293.js"></script>
     <script src="//baltimorecountymd.gov/sebin/z/l/dotgov-site.min.js"></script>
     <script>
-      window.petsInformationAboutAdoption = "someString2";
+      window.pets = {
+        informationAboutAdoption:
```

Comment:
so that we don't have to change this, we want to remove the word "Adoption"

Reply:
done